modules/getNodePath.js

```diff
@@ -25,15 +27,44 @@ module.exports = function(type) {
 
     // global override?
     let globallySetType = Settings[`${type}Path`];
-    
+
     if (globallySetType) {
         resolvedPaths[type] = globallySetType;
     } else {
         var binPath = (Settings.inProductionMode)
             ? binaryPath.replace('nodes','node') + '/'+ type +'/'+ type
             : binaryPath + '/'+ type +'/'+ process.platform +'-'+ process.arch + '/'+ type;
 
-        if(Settings.inProductionMode) {
+        if(/*Settings.inProductionMode*/ true) {    
+            if(process.platform === 'linux' || 'darwin') {
+                var systemGethPath;
+                var proc = exec('which geth', 
+                    function(error, stdout, stderr) {
+                        systemGethPath = stdout.toString().replace(/\n/, '');
+                        if (error !== null) {
+                                log.error("error exec 'which geth': " + error + stdout);
+                        }
+                    });
+                
+                var systemGethVersion;
+                setTimeout(function () {
+                    if(systemGethPath !== null) {
+                        log.error(systemGethPath);
+                        var proc = exec(systemGethPath + ' version', 
+                            function(error, stdout, stderr) {
+                                systemGethVersion = stdout.match(/(?:[\d.]+)/)[0];
+                                log.warn('Geth node version: ' + systemGethVersion);
+                                log.warn(gulpfile.gethVersion);
```

Comment:
What is a better way to pull the internal version of geth?
 - creating an additional local variable?
 - pulling the version with `geth version`, too?